android/src/main/java/com/adyenreactnativesdk/util/ReactNativeJson.kt

```diff
@@ -32,6 +32,8 @@ object ReactNativeJson {
                 map.putArray(key, convertJsonToArray(value))
             } else if (value is Boolean) {
                 map.putBoolean(key, value)
+            } else if (value is Long && (value as Long) < Int.MAX_VALUE) {
```

Comment:
I think `(value as Long)` is redundant because of `value is Long` before it, can you double check?

Reply:
Indeed :)
Fixed [3732d96](https://github.com/Adyen/adyen-react-native/pull/425/commits/3732d964adb1012a688d041bbc6e8cb6f463d80c)